src/helpers/SortPosts.js

```diff
@@ -0,0 +1,15 @@
+import { db } from "../firebaseConfig"
+
+export default async function sortPosts(sortBy) {
+  let allPosts = []
+
+  if (sortBy === "asen") {
+    const res = await db.collection("posts").orderBy("createdAt").get()
+    await res.forEach((res) => allPosts.push(res.data()))
+    return allPosts
+  }
+
+  const res = await db.collection("posts").orderBy("createdAt", sortBy).get()
```

Comment:
Conditional is not needed here, orderBy second argument can either be `asc` or `desc`. So change the sortBy argument to be "asc" instead of "asen".

So this can be rewritten as
```
export default async function sortPosts(sortBy) {
  let allPosts = []
  const res = await db.collection("posts").orderBy("createdAt", sortBy).get()
  await res.forEach((res) => allPosts.push(res.data()))
  return allPosts
}
```